package.json

```diff
@@ -69,25 +73,25 @@
     "@stylistic/eslint-plugin-ts": "^4.2.0",
     "@types/eslint": "^9.6.1",
     "@types/geojson": "^7946.0.16",
-    "@types/jest": "^29.5.14",
     "@types/node": "^22.15.17",
     "@typescript-eslint/eslint-plugin": "^8.32.0",
     "@typescript-eslint/parser": "^8.32.0",
+    "@vitest/coverage-v8": "3.1.3",
+    "@vitest/ui": "3.1.3",
     "dts-bundle-generator": "^9.5.1",
     "eslint": "^9.26.0",
-    "eslint-plugin-jest": "^28.11.0",
     "eslint-plugin-jsdoc": "^50.6.14",
+    "eslint-plugin-vitest": "^0.5.4",
     "glob": "^11.0.2",
     "globals": "^16.1.0",
-    "jest": "^29.7.0",
-    "jest-canvas-mock": "^2.5.2",
-    "jest-environment-jsdom": "^29.7.0",
+    "jsdom": "^26.1.0",
```

Comment:
Is this needed? I believe most functions should also work in node environment, so jsdom shouldn't be a dependency, shouldn't it?